test/extended/util/url/url.go

```diff
@@ -272,7 +330,13 @@ func (ut *Test) ToShell(i int) string {
 		}
 	}
 	lines = append(lines, `rc=0`)
-	cmd := fmt.Sprintf(`curl -X %s %s -s -S -o /tmp/body -D /tmp/headers %q`, ut.Req.Method, strings.Join(headers, " "), ut.Req.URL)
+	post := ""
+	cmd := ""
+	if strings.ToLower(strings.Trim(ut.Req.Method, " ")) == "post" {
+		cmd = `echo "` + ut.PostBody + `" > /tmp/post-body && `
+		post = " -H 'Expect:' -d @/tmp/post-body"
+	}
```

Comment:
`echo body | curl -d @-` will avoid the temp file clutter.
https://curl.haxx.se/mail/archive-2002-01/0093.html

Reply:
Yeah I tried that.  We've got enough funkiness in the various xml or groovy DSL snippets that the bash bombs out.

I tried encoding the body, no dice as well.

And the above I do have breaks down in some cases anyway.

I'm now looking at oc rsync'ing the files as is into the exec pod.  So far a `curl -d @filename` has been the only thing that holds up.  Just leads to a further refactor of the multitude of places we post data to jenkins, to make sure stuff is in a file.